### ISSUE TITLE
Chromium (Aura) context menus do not hold window focus.

### ISSUE THREAD

**Issue by Zach-Button** [swaywm/sway#1278]
**Steps to reproduce:**

1. Open Chromium or Vivaldi and position another window to the right of it as in:

```
==================================================
|                       |                        |
|   chromium            |                        |
|                       |                        |
|                       |                        |
|                       |                        |
|                       |                        |
==================================================
```

2. Near the right side of the chromium window, right click to open a context menu. Move your mouse cursor to position A

```
==================================================
|                       |                        |
|   chromium            |                        |
|                   ----------                   |
|                   |      A |                   |
|                   |        |                   |
|                   ----------                   |
==================================================
```

Upon leaving the boundry of the Chromium window, the other window takes focus (even though the mouse is over the context menu), and the context menu is closed.

**Expected behavior:** 

The chromium window retains focus as long as the context menu of the Chromium window is being hovered.

**Log:**

https://gist.github.com/Zach-Button/49cc069f56d96043ca076abb51391055
Version 0.13-rc1-37-gfd47a30

**Comment by Timidger** [comment]
Does disabling "focus on mouse over" fix your issue?

**Comment by Zach-Button** [comment]
The issue only occurs with focus on mouse over, because the issue occurs when the other window is focused. 

So the issue is related to "focus on mouse over" specifically.

**Comment by Zach-Button** [comment]
Issue updated with log and version.

**Comment by SirCmpwn** [comment]
I'm closing issues that are resolved by the yet-unreleased sway 1.0.
